vulture_os/services/frontend/form.py

```diff
@@ -75,17 +74,18 @@ class Meta:
         fields = ("enabled", "reputation_ctx", "arg_field", "dst_field")
 
         widgets = {
-            'enabled': CheckboxInput(attrs={'class': "form-control js-switch"}),
-            'reputation_ctx': Select(attrs={'class': "form-control select2"}),
-            'arg_field': TextInput(attrs={'class': "form-control", 'placeholder': "src_ip"}),
-            'dst_field': TextInput(attrs={'class': "form-control", 'placeholder': "src_reputation"})
+            'enabled': CheckboxInput(attrs={'class': 'form-control js-switch'}),
+            'reputation_ctx': Select(attrs={'class': 'form-control select2'}),
+            'arg_field': TextInput(attrs={'class': 'form-control', 'placeholder': 'src_ip'}),
+            'dst_field': TextInput(attrs={'class': 'form-control', 'placeholder': 'src_reputation'})
         }
 
     def __init__(self, *args, **kwargs):
         """ Initialisation of fields method """
         # Do not set id of html fields, that causes issues in JS/JQuery
         kwargs['auto_id'] = False
         super().__init__(*args, **kwargs)
+        self = bootstrap_tooltips(self)
         self.fields['reputation_ctx'].empty_label = None
 
```

Comment:
many changes not linked to the dev, please put them in a separate PR (bootstrap tooltips already seems in #504)